src/test/resources/mockdata/invoiceLines/invoice_line.json

```diff
@@ -1,39 +0,0 @@
-{
```

Comment:
This file is used by API tests a lot. Once this PR is merged, ideally API test updates should be ready to be merged as well